package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "build-examples": "yarn workspaces foreach -v --include 'example-*' -p run build-storybook",
+    "build-examples:overview": "bb",
     "test-ci": "yarn workspaces foreach  -v --exclude 'example-svelte' -p run test-ci",
```

Comment:
We need to `--exclude 'example-overview'` since it has no stories of its own to test.  That's why CI is failing right now.